iml-gui/crate/src/components/stratagem/mod.rs

```diff
@@ -394,22 +394,12 @@ pub(crate) fn view(model: &Model, all_locks: &Locks) -> Node<Msg> {
                 caption_wrapper(
                     "inode Usage Distribution",
                     Some(&last_scan),
-                    stratagem_chart(grafana_chart::GrafanaChartData {
-                        org_id: 1,
-                        refresh: "1m",
-                        panel_id: 2,
-                        vars: config.grafana_vars.clone()
-                    })
+                    stratagem_chart(grafana_chart::create_chart_params(2, "1m", config.grafana_vars.clone()))
```

Comment:
What is odd here now is the question why vars do not include `panel_id` and `refresh` :)